imjoy/__main__.py

```diff
@@ -106,7 +108,7 @@ def main():
             if distutils.spawn.find_executable("pip") is None:
                 git_cmd += " pip"
             if git_cmd != "":
-                logger.info("pip command failed, trying to install git and pip...")
+                print("pip command failed, trying to install git and pip...")
```

Comment:
I replaced logger with print, since we don't seem to have logging available here yet.